code/game/objects/items/devices/memorywiper.dm

```diff
@@ -67,7 +68,7 @@
 					plug(over_object)
 					visible_message("[usr] plugs \the [src] into \the [attached]'s maintenance port.")
 
-/obj/item/device/memorywiper/attack_hand(user as mob)
+/obj/item/device/memorywiper/attack_hand(mob/user)
 	if(attached)
 		to_chat(user, SPAN_NOTICE("You initialize the memory wipe protocols. This procedure will take approximately 30 seconds."))
 		to_chat(attached, SPAN_WARNING("The computer hums to life and you feel your memories bleed away into nothingness."))
```

Comment:
```suggestion
		to_chat(attached, SPAN_WARNING("The computer hums to life and you feel your memories begin to bleed away into nothingness."))
```
small suggestion - had this used on me once and felt like this first message was kinda ambiguous, wasn't sure if this was an in-progress thing or that the process was completed and i was supposed to be acting wiped already